HCCGo/app/index.html

```diff
@@ -15,7 +15,7 @@
     <link href="lib/bootstrap/css/bootstrap.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/material.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/ripples.css" rel="stylesheet">
-    <link href="lib/c3/c3.css" rel="stylesheet">
+    <link href="../node_modules/c3/c3.css" rel="stylesheet">
```

Comment:
Yes, these should remain available, as the hierarchy doesn't change after packaging. And it does seem a little odd but the directory above this one is where the node_modules folder needs to be.